yb-voyager/src/migassessment/sizing.go

```diff
@@ -190,31 +194,40 @@ func SizingAssessment() error {
 	shardedObjects, cumulativeIndexCountSharded :=
 		getListOfIndexesAlongWithObjects(finalSizingRecommendation.ShardedTables, sourceIndexMetadata)
 
-	// calculate time taken for colocated import
-	importTimeForColocatedObjects, parallelVoyagerJobsColocated, err :=
-		calculateTimeTakenAndParallelJobsForImportColocatedObjects(colocatedObjects,
-			finalSizingRecommendation.VCPUsPerInstance, finalSizingRecommendation.MemoryPerCore, experimentDB)
+	// get load times data from experimental database for sharded Tables
+	colocatedLoadTimes, err := getExpDataLoadTime(experimentDB, finalSizingRecommendation.VCPUsPerInstance,
+		finalSizingRecommendation.MemoryPerCore, COLOCATED_LOAD_TIME_TABLE)
 	if err != nil {
-		SizingReport.FailureReasoning = fmt.Sprintf("calculate time taken for colocated data import: %v", err)
-		return fmt.Errorf("calculate time taken for colocated data import: %w", err)
+		return fmt.Errorf("error while fetching sharded load time info: %w", err)
```

Comment:
should this error be for colocated table?

Reply:
done. made the changes.